src/VisualStudio/Core/Impl/Options/GridOptionPreviewControl.xaml

```diff
@@ -177,9 +175,10 @@
                 </DataGridTemplateColumn>
             </DataGrid.Columns>
         </DataGrid>
-        <GridSplitter Grid.Row="2" HorizontalAlignment="Stretch"></GridSplitter>
+        <GridSplitter Grid.Row="3" HorizontalAlignment="Stretch" Margin="0,28,0,7"/>
         <Border Grid.Row="3" BorderBrush="Gray" BorderThickness="1">
             <ContentControl Name="EditorControl" Content="{Binding TextViewHost, Mode=OneWay}" Focusable="False"></ContentControl>
         </Border>
+        <Button Content="Generate .editorconfig file from settings" HorizontalAlignment="Left" Margin="20,40,0,0" VerticalAlignment="Top" Width="235" Height="25" Click="Generate_Save_Editorconfig" />
```

Comment:
nit: since this is in the 0th row it would be nice if this was at the top so that the source order matches the ui

Reply:
also, why the left margin?

Reply:
nit: padding might be more appropriate than hard-coding with & height

Reply:
VerticalAlignment should be unnecessary?